.github/workflows/workflow.yml

```diff
@@ -8,63 +8,11 @@ on:
   pull_request:
 
 jobs:
-  sync_aea_loop_unit_tests:
-    continue-on-error: True
-    runs-on: ubuntu-latest
-    timeout-minutes: 30
-    steps:
-    - uses: actions/checkout@master
-    - uses: actions/setup-python@master
-      with:
-        python-version: 3.8
-    - uses: actions/setup-go@master
-      with:
-        go-version: '^1.14.0'
-    - name: Install dependencies (ubuntu-latest)
-      run: |
-        sudo apt-get update --fix-missing
-        sudo apt-get autoremove
-        sudo apt-get autoclean
-        pip install pipenv
-        pip install tox
-        sudo apt-get install -y protobuf-compiler
-    - name: Unit tests and coverage
-      run: |
-        tox -e py3.8 -- --aea-loop sync -m 'not integration and not unstable'
 
-  all_integrational_tests:
-    continue-on-error: True
+  common_checks_1:
```

Comment:
[minor comment] I would put a comment here to describe what's `common_checks_1` about (I presume you meant Python linting). Also, I would consider changing the name of the job to `python_linting`, or just `linting`, or similar.

Reply:
Its obvious from looking at the code/yaml spec; comments then always end up out of sync (see above ;))

Reply:
ok